spec/facility_spec.rb

```diff
@@ -1,4 +1,6 @@
 require 'spec_helper'
+require './lib/facility'
+require './lib/vehicle'
 
 RSpec.describe Facility do
   before(:each) do
```

Comment:
I would review all of your test in this spec file. Since we are in the `facility_spec` we want our `expect statements` to check for `facility` things. All of your `expect statements` for `registrant` will want to be in the `registrant_spec`. This iteration would require you to be making tests in both files.